### DISCUSSION TITLE
Weird Thing happen after flashing new firmware

### DISCUSSION THREAD

**Discussion by SlamberGamer** [Jyers/Marlin#1010] [Q&A] [answered]
Hi guys, I got a problem after i compiled my firmware.. please refer the video.. its very hard to translate what happened through the word.

[Youtube Video Link](https://www.youtube.com/watch?v=OhGhuK_Kn6c)
https://www.youtube.com/watch?v=OhGhuK_Kn6c

**Comment by Jyers** [comment, marked as answer]
Looks like it could be a hardware issue of some kind causing the mainboard to reboot. Is it still acting this way?
